sherlock: add page (#7606)
tldr-pages/tldr: pages/linux/sherlock.md

# sherlock

> Find usernames across social networks

> - Display help:
> `sherlock --help`

Reviewer: ```suggestion
- Display help:

`sherlock --help`
```

> - Save result to file (if using single username):
> `sherlock {{username}} --output {{filename}}`

Reviewer: ```suggestion
- Save result to file (if using single username):

`sherlock {{username}} --output {{filename}}`
```

> - Save result to folder (if using multiple usernames):
> `sherlock {{[usernames]}} --folderoutput {{foldername}}`

Reviewer: ```suggestion
- Save result to a folder (if using multiple usernames):

`sherlock {{username1 username2 ...}} --folderoutput {{path/to/directory}}`
```

> - Make requests over Tor:
> `sherlock --tor {{username}}`

Reviewer: ```suggestion
- Make requests over Tor:

`sherlock --tor {{username}}`
```

> - Make requests over Tor with new Tor circuit after each request:
> `sherlock --unique-tor {{username}}`

Reviewer: ```suggestion
- Make requests over Tor with new Tor circuit after each request:

`sherlock --unique-tor {{username}}`
```

> - Make requests over a proxy:
> `sherlock {{username}} --proxy {{proxyurl}}`

Reviewer: ```suggestion
- Make requests over a proxy:

`sherlock {{username}} --proxy {{proxyurl}}`
```

> - Browse to all results on default browser:
> `sherlock {{username}} --browse`

Reviewer: ```suggestion
- Browse to all results on default browser:

`sherlock {{username}} --browse`
```
